java/code/webapp/WEB-INF/pages/common/fragments/configuration/files/contents.jspf

```diff
@@ -28,7 +28,7 @@
 
                         <strong><bean:message key="filedetails.contents.jspf.download-header"/></strong><br />
                         <bean:message key="filedetails.contents.jspf.download"
-                                arg0="/rhn/configuration/file/FileDownload.do?crid=${revision.id}&amp;cfid=${revision.configFile.id}"
+                                arg0="/rhn/configuration/file/FileDownload.do?crid=${revision.id}&amp;cfid=${revision.configFile.id}\" data-senna-off=\"true"
```

Comment:
Shouldn't we add the `data-senna-off="true"` on `StringResource_*.xml` files directly, instead of injecting it here?

Reply:
`data-senna-off="true"` was now moved to `StringResource_en_US.xml`.